Cargo.toml

```diff
@@ -64,6 +64,10 @@ path = "sdk/identity/azure_identity"
 version = "0.1.0"
 path = "sdk/storage"
 
+[workspace.dependencies.azure_storage_blob]
+version = "0.1.0"
+path = "sdk/storage/azure_storage_blob"
+
```

Comment:
You should only add this for other crates outside your service directory.

Reply:
Within your service directory, just use a `path` dependency like I did for `sdk/keyvault`.